Fix - add textPath svg element

liorEhrlich opened the pull request:
The svg element textPath was missing from the list of dom elements. 
Added it to the domElements, and to v4's domElements.

probablyup commented:
@liorEhrlich thanks, could you change this PR to target the `legacy-v5` branch and add a changelog entry?

liorEhrlich commented:
@probablyup Thanks for reviewing my PR.
I've added a changelog entry, but had some issues changing the parent branch of the feature branch.
Moved the code to: https://github.com/styled-components/styled-components/pull/3461
